codespell_lib/data/dictionary.txt

```diff
@@ -8134,6 +8150,7 @@ ourselfs->ourselves
 ourselve->ourselves
 ourselvs->ourselves
 oustanding->outstanding
+out-of-bound->out-of-band
```

Comment:
could also very well be `out-of-bounds`. I haven't encountered `out-of-band` before. Not sure about this one.

Reply:
Kernel related.
Read here:
https://en.wikipedia.org/wiki/Out-of-band_management

Anyway, I'm going to remove it.